packages/eslint-plugin-gestalt/src/no-spread-props.test.js

```diff
@@ -0,0 +1,67 @@
+// @flow strict
```

Comment:
Next on our list of "create our own AST framework" work: consolidating how we write these test files. So much boilerplate. It'd be especially awesome if we establish filename/directory conventions around valid and invalid tests, such that we no longer have to individually list the input files and their corresponding outputs.

Reply:
Agree. I didn't look into this BUT I felt the pain!

Reply:
So worth exploring.